main.go

```diff
@@ -85,6 +85,15 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+  // Stop users giving themselves karma.
+  if author == user {
+    amount = decreaseAmount(text)
+    s.Decrease(user, amount)
+    response.Text = author + " cheated themselves out of " + strconv.Itoa(amount) + " karma"
+    response.Send(*cliCallback)
+    return
+  }
+
```

Comment:
inorite? I take no responsibility for all those tabs, I cannot even add them in my editor, as it converts them to two spaces the second I hit the tab key.

As for inconsistency ... it's go. It's supposed to be inconsistent, it's what all the cool kids do now.